plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -142,6 +176,11 @@ public List<SchemaTableName> listTables(ConnectorSession session, Optional<Strin
     public Map<String, ColumnHandle> getColumnHandles(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
         MongoTableHandle table = (MongoTableHandle) tableHandle;
+
```

Comment:
Since we capture FILTER to be applied on a collection - can we reuse the same columns of that of a collection ?